src/XRP/common-xrpl-client-interface.ts

```diff
@@ -0,0 +1,39 @@
+import { Wallet, XrplNetwork } from 'xpring-common-js'
+import TransactionResult from './model/transaction-result'
+import RawTransactionStatus from './raw-transaction-status'
+
+/**
+ * An interface describing CommonXrplClient.
+ */
+export default interface CommonXrplClientInterface {
```

Comment:
good q, it does not because the only reason those methods had to be in the `XrpClientInterface` in the first place was because they were needed in the reliable submission logic.  So the `ReliableSubmissionXrpClient` needed to have them to use them in its interpretation of that reliable logic.  Now that those methods and the reliable submission logic itself all live in the same class, we don't need to guarantee that they span several client classes via an interface.